### PR TITLE
Mvella1/first update

### PR DESCRIPTION
**Pull request by mvella1**
updates to meetings 11/19/19 through 2/25/20

### CODE REVIEW

File: _events/1.markdown [hunk 1]
```diff
@@ -1,12 +1,26 @@
...
 layout: event
 title:  "2019-07-02 Call"
-date:   2019-07-02 9:00
-file_list:
-  # google drive links
-  test.png: Praesent sed mi at orci pretium hendrerit. Vestibulum pharetra laoreet mi id maximus.
-  https://www.foo.com: Aenean accumsan consequat augue sit amet porttitor.
-summary: This is a summary of the call.
+date:   2019-07-02 10:00
+file_list:HCA metadata schema: https://github.com/HumanCellAtlas/metadata-schema
```
Comment (ilan-gold) [inline _events/1.markdown]:
```
file_list:
  HCA metadata schema: https://github.com/HumanCellAtlas/metadata-schema
  HCA schema validator: https://github.com/HumanCellAtlas/ingest-validator-js
 .
 .
 .
```

I think it'd be more readable (and functional) to make this look like https://raw.githubusercontent.com/hms-dbmi/x-consortium/master/_example_templates/0101.markdown i.e an indented list under `file_list`

File: _events/1.markdown [hunk 2]
```diff
@@ -19,3 +33,76 @@ summary: This is a summary of the call.
...
+- How long to fill out metadata/annotations?
+  - Hard to say, varies by # samples and experiment complexity, generally doesn’t take too long, file upload is the longest step
+- Any APIs to submit? No
+- Is it open source (MAGE-TAB, Annotare)? Yes
+    - Annotare on GitHub: https://github.com/arrayexpress/annotare2
+    - Annotare's built-in MAGE-TAB validation: https://github.com/arrayexpress/magetabcheck
+    - MAGE-TAB spec: http://fged.org/projects/mage-tab/
+
+
+4DN (4D Nucleome) Metadata Tools
```
Comment (ilan-gold) [inline _events/1.markdown]: Should this be a header?

File: _events/1.markdown [hunk 3]
```diff
@@ -19,3 +33,76 @@ summary: This is a summary of the call.
...
+Other efforts that should be contacted:
+Rebuilding a Kidney (RBK) and GenitoUrinary Development Molecular Anatomy Project (GUDMAP): About 9 years into developing data and portal. NIH funded. Extensive meta-data and data specifications including scRNAseq, imaging etc. Data ingestion pipeline. Focused on the Genitourinary tract, including the kidney. https://www.gudmap.org/about/
+
+Kidney Precision Medicine Project (KPMP). NIH funded.Several years into project. Focused on the kidney. https://kpmp.org
+
+ApiNATOMY (https://bivi.co/visualisation/apinatomy). Contact: Dr. Bernard de Bono. Data and meta-data specifications, data pipeline.
+
+9/7/2019 follow-up call:
+
+ArrayExpress/scExpression Atlas
```
Comment (ilan-gold) [inline _events/1.markdown]: Should this be a header?

File: _events/1.markdown [hunk 4]
```diff
@@ -19,3 +33,76 @@ summary: This is a summary of the call.
...
+
+Other efforts that should be contacted:
+Rebuilding a Kidney (RBK) and GenitoUrinary Development Molecular Anatomy Project (GUDMAP): About 9 years into developing data and portal. NIH funded. Extensive meta-data and data specifications including scRNAseq, imaging etc. Data ingestion pipeline. Focused on the Genitourinary tract, including the kidney. https://www.gudmap.org/about/
+
+Kidney Precision Medicine Project (KPMP). NIH funded.Several years into project. Focused on the kidney. https://kpmp.org
+
+ApiNATOMY (https://bivi.co/visualisation/apinatomy). Contact: Dr. Bernard de Bono. Data and meta-data specifications, data pipeline.
+
+9/7/2019 follow-up call:
```
Comment (ilan-gold) [inline _events/1.markdown:84]: Should this be a (larger) header?

File: _events/5.markdown [hunk 5]
```diff
@@ -1,18 +1,23 @@
 ---
 layout: event
 title:  "2019-11-19 Call"
-date:   2019-11-19 9:00
+date:   2019-11-19 10:00
 file_list:
-  # google drive links
-summary: This is a summary of the call.
+- https://docs.google.com/presentation/d/18bT4-wVe3PheeiMLKRvQxpXoCeSjzH3Hj-Axn6V2VFE/edit#slide=id.g78fcc90ba3_2_110
```
Comment (ilan-gold) [inline _events/5.markdown]: We don't need the dash `-`, just the indent like in https://raw.githubusercontent.com/hms-dbmi/x-consortium/master/_example_templates/0101.markdown

File: _events/6.markdown [hunk 6]
```diff
@@ -1,10 +1,10 @@
 ---
 layout: event
 title:  "2019-12-03 Call"
-date:   2019-12-03 9:00
+date:   2019-12-03 10:00
 file_list:
-  # google drive links
-summary: Discussion of meta data use cases.
+- https://docs.google.com/presentation/d/1JZtv_q-fgXBoi1efw-VSJvQYCBEPx-SwehaRnLCQLTM/edit?usp=sharing
```
Comment (ilan-gold) [inline _events/6.markdown]: We don't need the dash -, just the indent like in https://raw.githubusercontent.com/hms-dbmi/x-consortium/master/_example_templates/0101.markdown

**Review by ilan-gold — COMMENTED** [review]
Should be pretty quick changes.  Looks good otherwise!

File: _events/6.markdown (hunk 6 above)
Comment (ilan-gold) [reply _events/6.markdown]: @mvella1 i think this is still here

File: _events/1.markdown [hunk 7]
```diff
@@ -1,12 +1,19 @@
...
+https://docs.google.com/presentation/d/1fExLowEL5jHoWc1AloepP_7n6GWwg6TZ4bl1iKleI8I/edit#slide=id.g5c7728c05e_0_11: Presentation of HCA metadata principles
+https://data.4dnucleome.org/profiles/: 4DN metadata schemas (To view any particular schema expand with + and click on the ID link.Add /?format=json to get the raw json view; Structure and validation very similar to what was presented by Jason from ENCODE; ENCODE metadata schemas (presented by Jason Hilton jahilton@stanford.edu)
+https://github.com/ENCODE-DCC/encoded/tree/dev/src/encoded/schemas: Raw
+https://www.encodeproject.org/profiles/: More human-friendly
+https://github.com/ENCODE-DCC/encoded/tree/dev/src/encoded/audit: Post-indexing audits
+https://academic.oup.com/database/article/doi/10.1093/database/baw001/2630135: 2016 paper
+https://docs.google.com/presentation/d/1HBvbw8aSZzozm_FoMtbVUVMK9SeTBGKhGbNZBWxvQ8U/edit?usp=sharing: HTAN metadata presentation
+https://drive.google.com/a/ebi.ac.uk/file/d/16bZFR1OGCufy2-eb4XkErONtpZK9MtGZ/view?usp=sharing: AE metadata presentation
+summary: A discussion of metadata schema specifications, metadata validation, and ontology validation.
```
Comment (ilan-gold) [inline _events/1.markdown]: is this indented? i think it still needs be

File: _events/5.markdown (hunk 5 above)
Comment (ilan-gold) [reply _events/5.markdown]: i don't think the dash is necessary still

File: _events/6.markdown (hunk 6 above)
Comment (ilan-gold) [reply _events/6.markdown]: i don't think the dash is necessary still

File: _events/4.markdown [hunk 8]
```diff
@@ -1,15 +1,28 @@
 ---
 layout: event
 title:  "2019-10-08 Call"
-date:   2019-10-08 9:00
+date:   2019-10-08 10:00
 file_list:
-  # google drive links
-summary: This is a summary of the call.
+- (https://docs.google.com/presentation/d/1toZITnCCHTS7yoqODjop6qtI17CHCiyEJLInREQJSsc/edit#slide=id.g62e11bdf07_0_0)
```
Comment (ilan-gold) [inline _events/4.markdown]: I think you need to remove the parentheses and also the dash.  Additionally, you need a description (like in the _example_templates) folder of each link.

File: _events/6.markdown [hunk 9]
```diff
@@ -1,10 +1,11 @@
 ---
 layout: event
 title:  "2019-12-03 Call"
-date:   2019-12-03 9:00
+date:   2019-12-03 10:00
 file_list:
-  # google drive links
-summary: Discussion of meta data use cases.
+  https://docs.google.com/presentation/d/1JZtv_q-fgXBoi1efw-VSJvQYCBEPx-SwehaRnLCQLTM/edit?usp=sharing
```
Comment (ilan-gold) [inline _events/6.markdown]: Needs description

File: _events/5.markdown [hunk 10]
```diff
@@ -1,18 +1,23 @@
 ---
 layout: event
 title:  "2019-11-19 Call"
-date:   2019-11-19 9:00
+date:   2019-11-19 10:00
 file_list:
-  # google drive links
-summary: This is a summary of the call.
+  https://docs.google.com/presentation/d/18bT4-wVe3PheeiMLKRvQxpXoCeSjzH3Hj-Axn6V2VFE/edit#slide=id.g78fcc90ba3_2_110
```
Comment (ilan-gold) [inline _events/5.markdown]: Needs description

**Review by ilan-gold — APPROVED** [review]

**Comment by mvella1** [comment]
Updated all past meeting content and added in future scheduled topics

**Pull request merged**